jekyll/_cci2/non-aws.md

```diff
@@ -68,7 +68,7 @@ Each machine on which the Nomad Clients are installed reserves two CPUs and 4GB
 ### GitHub
 
 CircleCI uses GitHub or GitHub Enterprise credentials for
-authentication which, in turn, may use LDAP, SAML, or SSH for access. That is, CircleCI will inherit the authentication supported by your central SSO infrastructure. The following table describes the ports used on machines running GitHub to communicate with the Services and Nomad client instances.
+authentication which, in turn, may use LDAP, SAML, or SSH for access; that is, CircleCI will inherit the authentication supported by your central SSO infrastructure. The following table describes the ports used on machines running GitHub to communicate with the Services and Nomad client instances.
```

Comment:
Rather than [adding a semicolon](https://github.com/circleci/circleci-docs/wiki/Style-Guide#dashes-and-semicolons), perhaps this is an opportunity to rewrite this sentence? The leading "that is" is a writing smell to me; that is, it makes me think that something wasn't communicated clearly _before_ the semicolon. :)